examples/hello/tsconfig.json

```diff
@@ -4,7 +4,7 @@
     "target": "esnext",                                  /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */
     "experimentalDecorators": true,                      /* Enable experimental support for legacy experimental decorators. */
     "emitDecoratorMetadata": true,                       /* Emit design-type metadata for decorated declarations in source files. */
-    "module": "commonjs",                                /* Specify what module code is generated. */
+    "module": "Node16",                                /* Specify what module code is generated. */
```

Comment:
Is this the correct setting for a DBOS app?

Reply:
That's what the TS docs recommend. Their view of our current setting, `commonjs` is, ["don't use this"](https://www.typescriptlang.org/docs/handbook/modules/reference.html#commonjs).

Reply:
Changing this tsconfig will also impact `dbos init`. I agree with "not commonjs" but the setting for a library like dbos-sdk may not be the right setting for an app like `examples/hello`

Reply:
We ought to split this app out.  But wonder if this one line needs to go in at the same time as the change for CLI?

Reply:
I don't think it does, this is a somewhat orthogonal change. Can test that. If we don't used `Node16`, what should we use?